src/markdown-pages/build-apps/publish-deploy.mdx

```diff
@@ -32,11 +32,11 @@ From the New Relic One **Apps** page, you can review available apps and subscrib
 
 Here, you learn to:
 
-- Serve your app locally
-- Add images and metadata to your app
-- Publish it
-- Subscribe and unsubscribe accounts you manage to the app
-- Handle duplicate applications
+- [Serve your app locally](#Serve-your-app-locally)
+- [Add images and metadata to your app](#Add-images-and-metadata-to-your-apps)
+- [Publish it](#Publish-your-app)
+- [Subscribe and unsubscribe accounts you manage to the app](#Subscribe-or-unsubsribe-apps)
+- [Handle duplicate applications](#Handle-duplicate-applications)
```

Comment:
```suggestion
- [Serve your app locally](#serve-your-app-locally)
- [Add images and metadata to your app](#add-images-and-metadata-to-your-apps)
- [Publish it](#publish-your-app)
- [Subscribe and unsubscribe accounts you manage to the app](#subscribe-or-unsubsribe-apps)
- [Handle duplicate applications](#handle-duplicate-applications)
```

It looks like the anchor is case senstive so these need to be lower case. The `id` attribute that is generated for each header (which is done automatically for us by a plugin) will always be lower case.

Reply:
Thanks. I was wondering about that.